packages/okta-vue/src/utils/Utils.js

```diff
@@ -0,0 +1,22 @@
+const initConfig = auth => {
+  const missing = []
+  if (!auth.issuer) missing.push('issuer')
+  if (!auth.client_id) missing.push('client_id')
+  if (!auth.redirect_uri) missing.push('redirect_uri')
+  if (!auth.scopes) auth.scopes = ['openid']
+  if (missing.length) throw new Error(`${missing.join(', ')} must be defined`)
+  return auth
+}
+
+const getFromUri = () => {
+  const path = localStorage.getItem('referrerPath') || '/'
```

Comment:
"referrer" feels a bit overloaded.  I feel like we should be more descriptive, something like `postLoginPath`

Reply:
This is a design decision we made earlier to be used across all of our SDKs. Noted [in this comment](https://github.com/okta/okta-oidc-js/pull/11#discussion-diff-141099927R26).

We can create a follow-up issue to change this if you'd like, but this value shouldn't be used/modified by the end developer.